website/profile/views.py

```diff
@@ -753,16 +754,21 @@ def unserialize_school(school):
 def unserialize_contents(field, func, auth):
```

Comment:
Is this function called when profile info is updated via APIv2?

Reply:
It's not, only when making `PUT` requests with the current API v1 route `settings/schools/` or `settings/jobs/`, which is what the current frontend does, hence the double spam checks!

Reply:
I guess I'm really asking if we're checking for spam when profile content is updated through APIv2. If not, that should be added to this PR so we don't need to implement it later when that page is emberized.

Reply:
Very belated answer to this question: yes!

Check happens in [user serializer here](https://github.com/CenterForOpenScience/osf.io/pull/8784/files#diff-4bd3e0861a11a2b856682206b4ecea42R218) and [test is here](https://github.com/CenterForOpenScience/osf.io/pull/8784/files#diff-2507ed7c57308a32c0327532a858505dR1146)